packages/flutter/lib/src/widgets/heroes.dart

```diff
@@ -152,10 +216,10 @@ class Hero extends StatefulWidget {
     this.placeholderBuilder,
     this.transitionOnUserGestures = false,
     required this.child,
-  }) : assert(tag != null),
-       assert(transitionOnUserGestures != null),
-       assert(child != null),
-       super(key: key);
+  })   : assert(tag != null),
```

Comment:
revert all these unrelated formatting changes! (here and below)

Reply:
Done Thanks